editor/source/assets/ManifestSerializer.cpp

```diff
@@ -0,0 +1,128 @@
+#include "ManifestSerializer.h"
+#include "AssetManifest.h"
+#include "utility/Output.h"
+
+#include <fstream>
+
+namespace
+{
+    const auto AudioClipTag = std::string{"AudioClip"};
+    const auto ConcaveColliderTag = std::string{"ConcaveCollider"};
+    const auto HullColliderTag = std::string{"HullCollider"};
+    const auto MeshTag = std::string{"Mesh"};
+    const auto TextureTag = std::string{"Texture"};
+
+    auto ReadStandardAsset(std::ifstream& file) -> nc::editor::Asset
```

Comment:
"Standard" here implies IMO that its one of our built-in assets. I would name this ReadCustomAsset or something